pkg/app/app.go

```diff
@@ -290,3 +298,9 @@ func (a App) HasImageOrImgpkgBundle() bool {
 	}
 	return false
 }
+
+func (a App) isNamespaceTerminating() bool {
```

Comment:
The simplest thing is probably to move this to ComponentInfo. The name doesn't fully reflect it's intention. It's intention is to abstract away calls to the cluster for things outside the scope of the factories. It was introduced so that we don't pass the coreClient around in this way.

Reply:
This makes so much sense! I was trying to find a way of not having to pass the coreClient. Thank you!!

Reply:
Done